app/views/shared/_bootstrap_tree.html.haml

```diff
@@ -0,0 +1,7 @@
+#bootstrap-tree{:style => "padding-left: 17px; padding-right: 30px", "ng-controller" => "bootstrapTreeController"}
```

Comment:
wondering about this `#bootstrap_tree` .. are you planning to be using this approach from more than one place (in which case bootstrap tree is a generic thing that should be a class so that it can happen more than once), or is this a one time thing? (in which case maybe a more descriptive id would be in order?)